src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -235,7 +228,7 @@ public LogUnitServer(ServerContext serverContext) {
             // support sparse files.  If we use the default 2GB file size, then
             // every time that a sparse file is closed, the OS will always
             // write 2GB of data to disk.  {sadpanda}  Use this static class
-            // var to signal to RollingLog to use a smaller file size.
+            // var to signal to StreamLogFiles to use a smaller file size.
             maxLogFileSize = 4_000_000;
         }
 
```

Comment:
On OS X, the default huge file size is a serious test hinderance.  Today's JUnit tests are almost always using `InMemoryLog` and therefore don't generate any disk state.  For any test that uses `RollingLog`, the simple act of writing a single log entry and then closing the log will result in the OS flushing 2GB of disk.  On a good day, my MBP can write 400MByte/sec, so that "only" delays a few seconds.  On a bad day, my MBP is doing wear leveling and whatever else, and this flush can take 10 seconds.  And that's when run on a system with a mostly-very-fast SSD and not a Winchester disk.